bower.json

```diff
@@ -84,8 +84,11 @@
   },
   "resolutions": {
     "moment": "~2.18.1",
-    "patternfly": "~3.24.0",
-    "jquery": "2.1.4",
-    "font-awesome": "~4.7.0"
+    "patternfly": "~3.25.1",
+    "jquery": "3.2.1",
+    "font-awesome": "~4.7.0",
+    "lodash": "4.x",
```

Comment:
I think this was due to pulling in the current origin-web-common.   Once the common PR is merged and released I'll try again without the resolution.